sdk/storage/azure-storage-files-datalake/test/ut/datalake_file_system_client_test.cpp

```diff
@@ -397,6 +398,133 @@ namespace Azure { namespace Storage { namespace Test {
     }
   }
 
+  TEST_F(DataLakeFileSystemClientTest, CustomerProvidedKey_LIVEONLY_)
+  {
+    // will skip test under some cased where test can't run (usually LIVE only tests)
```

Comment:
Why is this live-only?

Reply:
I recorded and find that the google test can't record all the request. Just create/delete filesystem requests were recorded. So playback failed.